public/pages/workflow_detail/agentic_search/test_flow/test_flow.tsx

```diff
@@ -161,24 +176,21 @@ export function TestFlow(props: TestFlowProps) {
               overflowX: 'hidden',
             }}
           >
-            <EuiPanel color="subdued" paddingSize="s">
+            <EuiPanel paddingSize="none" hasBorder={false} hasShadow={false}>
               <EuiFlexGroup direction="column" gutterSize="m">
+                {!opensearchLoading && Object.values(indices)?.length === 0 && (
```

Comment:
can we have a const to check if indices are present? Otherwise it can be undefined as well

Reply:
Sure. Although technically this value can't be undefined as it's a controlled redux var (although it got corrupt from bugs in the past), so I'll add for robustness.